filesystem/azure.go

```diff
@@ -212,7 +212,7 @@ func (f *AzureFile) Write(b []byte) (int, error) {
 		return 0, err
 	}
 	if err == io.EOF {
-		return 0, fmt.Errorf("Need blob content")
+		return 0, fmt.Errorf("Need write content")
```

Comment:
Also, you can check the size of the `b` before this (len(b) > chunksize). So we can avoid this special case checking.